depends/funcs.mk

```diff
@@ -194,13 +194,13 @@ $($(1)_preprocessed): | $($(1)_extracted)
 $($(1)_configured): | $($(1)_dependencies) $($(1)_preprocessed)
 	$(AT)echo Configuring $(1)...
 	$(AT)rm -rf $(host_prefix); mkdir -p $(host_prefix)/lib; cd $(host_prefix); $(foreach package,$($(1)_all_dependencies), tar --no-same-owner -xf $($(package)_cached); )
-	$(AT)mkdir -p $$(@D)
-	$(AT)+cd $$(@D); $($(1)_config_env) $(call $(1)_config_cmds, $(1))
+	$(AT)+mkdir -p $$(@D)
```

Comment:
This seems to be the only `+` left after your suggested changes (which look good to me, fixing `cd`ing into directories that have not been created). Wouldn't it be possible/better to drop this last `+`, too? Or is there a reasoning why this line has to be prefixed with a `+`, creating a folder?

To be honest: I've not run `make` to test it out and I'm new to the code base, so just asking.

Reply:
@madietlx

> Or is there a reasoning why this line has to be prefixed with a `+`, creating a folder?

Not-creating that folder causes an error when one runs `make --dry-run`. The `+` forces `make` to execute a line even in `--dry-run` mode.

Reply:
> Not-creating that folder causes an error when one runs `make --dry-run`. The `+` forces `make` to execute a line even in `--dry-run` mode.

@hebasto And that error would still occur - even though you did remove the `+` in front of the following `cd` line which caused the error in the first place?

Reply:
Thanks. [Updated](https://github.com/bitcoin/bitcoin/pull/19891#issuecomment-691027367).